Core/Frameworks/Baikal/Model/Config/Standard.php

```diff
@@ -116,7 +188,7 @@ function label() {
     }
 
     function set($sProp, $sValue) {
-        if ($sProp === "admin_passwordhash" || $sProp === "admin_passwordhash_confirm") {
+        if ($sProp === "admin_passwordhash" || $sProp === "admin_passwordhash_confirm" || $sProp == "ldap_bind_password") {
```

Comment:
Like this, the `ldap_bind_password` is not saved in the config file. There is no way to set a password from the admin interface.